packages/slate-renderer/src/elements/Embed/lib/injectOembedMarkup.ts

```diff
@@ -13,12 +13,20 @@ const injectOembedMarkup = ({ oembed, onError, target }: Parameters): void => {
     container.innerHTML = oembed.html || '';
     const embedScripts = Array.from(container.getElementsByTagName('script'));
 
+    const currentScriptSources = Array.from(document.getElementsByTagName('script')).map((script) =>
```

Comment:
_this comment is in an unrelated place_

https://linear.app/prezly/issue/DEV-2184#comment-826b8e56

>  Though I couldn't find a way to test that it indeed works. I believe I will have to wait for @kamilmielnik to return and help with that

Yes, there is no easy way to test it. I've created an issue for it: https://github.com/prezly/slate/issues/21

What you can do now is:
1. Set up the `slate` repository locally
2. Make sure to build it
3. Use local path (`file:`) in [`package.json` of your project](https://github.com/prezly/website-nextjs/blob/main/package.json#L14) to specify path to `@prezly/slate-renderer` package you've built locally - see [npm docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#local-paths)

Reply:
Thanks for a tip, will try to test it like that!